src/WebUI/Common/BaseKubeTable.scss

```diff
@@ -28,6 +20,10 @@
         .kind-tag {
             margin-right: 8px;
         }
+        
+        .bolt-card-header {
```

Comment:
we should see a better rather override, even if we do, we should have a seperate file, so later we can fix all at once.